cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -86,6 +87,20 @@ std::unique_ptr<table> distinct(
   rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @copydoc cudf::distinct(table_view const&, std::vector<size_type> const&, duplicate_keep_option,
+ *                         null_equality, rmm::cuda_stream_view, rmm::mr::device_memory_resource*)
+ *
+ * @param[in] stream CUDA stream used for device memory operations and kernel launches.
+ */
+std::unique_ptr<table> distinct(
+  table_view const& input,
+  std::vector<size_type> const& keys,
```

Comment:
Though the input type is changed, it shouldn't break any existing code.

Reply:
Wait, this will cascade compile error into `table_view::select(std::vector<size_type> const& column_indices) const`. Changing this function would further cause more breaking changes.

Reply:
I have filed an issue for that: https://github.com/rapidsai/cudf/issues/11116

Reply:
:+1: Thanks!